src/game_battlealgorithm.cpp

```diff
@@ -987,10 +1004,42 @@ bool Game_BattleAlgorithm::Normal::Execute() {
 
 		int effect = (source->GetAtk() / 2 - GetTarget()->GetDef() / 4);
 
+		// Row attacker adjustments (RPG2k3 only)
+		if (Player::IsRPG2k3()) {
+			if (Game_Battle::GetBattleCondition() == lcf::rpg::System::BattleCondition_none || Game_Battle::GetBattleCondition() == lcf::rpg::System::BattleCondition_initiative) {
+				if (source->GetType() == Game_Battler::Type_Ally && static_cast<Game_Actor*>(source)->GetBattleRow() == Game_Actor::RowType::RowType_front) {
+					effect = 125 * effect / 100;
+				}
+			}
+			if (Game_Battle::GetBattleCondition() == lcf::rpg::System::BattleCondition_surround) {
```

Comment:
You forgot the back/pincer  back row case

Reply:
This is because the pincer battle condition doesn't do any changes at all. More details about that in my comments in #680.

Reply:
As we discussed, you're right about pincer. But this is still missing back attack case.

Reply:
Whoops, after doing some retests you are right about the back attack case. I will fix this in the next commit.